src/main/kotlin/no/digdir/fdk/searchservice/service/SearchService.kt

```diff
@@ -169,5 +170,23 @@ class SearchService(
         return queryFilters
     }
 
+    private fun QueryFields.exactPaths(): List<String> =
+        listOf(
+            if (title) languagePaths("title", 30)
```

Comment:
Reduser denne og så har vi heller en veldig høy boost på eksakt treff
```suggestion
            if (title) languagePaths("title", 15)
```